### ISSUE TITLE
initialize the tracking with bounding box

### ISSUE THREAD

**Issue by senemaktas** [yformer/EfficientTAM#18]
Hi,

Thank you for providing the project.

I am ready to start tracking with the given first object point. However, I would like to initialize the tracking using bounding boxes instead.

I have attempted to define the bounding box in the following format: (xmin, ymin, xmax, ymax), with the label set to 1. I have also tried both commented and uncommented versions of the "points" parameter, but unfortunately, it does not seem to work.

Could you please provide an example of how to properly use a bounding box in this context? 

Here is the code snippet I am working with:

`
first_obj_bbox_xyxy = np.array(first_obj_bbox_xyxy, dtype=np.float32)
labels = np.array([1], np.int32)
prompts[ann_obj_id] = first_obj_bbox_xyxy, labels

_, out_obj_ids, out_mask_logits = predictor.add_new_points_or_box(
    inference_state=inference_state,
    frame_idx=ann_frame_idx,
    obj_id=ann_obj_id,
    # box=first_obj_bbox_xyxy,
    points=points,
    labels=labels,
)
`

Thanks,

**Comment by okideal** [comment]
@senemaktas 
try this dude [segment-anything-2-real-time](https://github.com/Gy920/segment-anything-2-real-time/blob/main/demo/demo.py)

**Comment by senemaktas** [comment]
Thanks very much, the below code work: 
`   
```
    # Let's add a box at (x_min, y_min, x_max, y_max) = (300, 0, 500, 400) to get started
    first_obj_bbox_xyxy = np.array([50, 0, 100, 150], dtype=np.float32)
    _, out_obj_ids, out_mask_logits = predictor.add_new_points_or_box(
        inference_state=inference_state,
        frame_idx=ann_frame_idx,
        obj_id=ann_obj_id,
        box=first_obj_bbox_xyxy
    )
    # labels = np.array([1], np.int32)
    # prompts[ann_obj_id] = first_obj_bbox_xyxy, labels
```
`
